server/appengine/src/test/java/who/RefreshCaseStatsServletTest.java

```diff
@@ -51,4 +49,29 @@ public void testParse() throws UnsupportedEncodingException {
     );
     assertEquals(113, s1.dailyCases.intValue());
   }
+
+  @Test
+  public void testDec24Data() throws IOException {
+    RefreshCaseStatsServlet servlet = new RefreshCaseStatsServlet();
+
+    InputStream inCompressed =
+      this.getClass()
+        .getClassLoader()
+        .getResourceAsStream("who-data-24dec2020.json.gz");
+    GZIPInputStream decompressed = new GZIPInputStream(inCompressed);
+    JsonObject root = JsonParser
+      .parseReader(new InputStreamReader(decompressed, "UTF-8"))
+      .getAsJsonObject();
+    JsonArray rows = root.getAsJsonArray("features");
+
+    RefreshCaseStatsServlet.JurisdictionData globalData = new RefreshCaseStatsServlet.JurisdictionData();
+    Map<String, RefreshCaseStatsServlet.JurisdictionData> countryData = new HashMap<>();
+
+    servlet.processWhoStats(rows, countryData, globalData);
+
+    assertEquals(1608768000000L, globalData.lastUpdated);
+    // The TotalCases is a good heuristic that the data loaded correctly: it is a sum
+    // of every daily case seen in the system so far.
+    assertEquals(77530799L, globalData.totalCases);
```

Comment:
This is a good one... but you add more. I was about to do add the same test:

<img width="581" alt="Screen Shot 2020-12-25 at 9 08 38 PM" src="https://user-images.githubusercontent.com/679339/103145837-5f049600-46f5-11eb-9201-cecf398603e2.png">

Reply:
if you send me some text i can put those in?